client/src/components/NewsSlider/index.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react';
+import hoverArrow from '../../assets/img/news-hover-arrow.svg';
+import { Swipe } from 'react-swipe-component';
+import articles from '../../assets/data/slider';
+import './style.scss';
+
+let startArray = [];
+
+if (articles.length < 4) {
+  articles.forEach((item, index)=>{
+    startArray.push(index + 1);
+  });
+} else {
+  startArray = [ 1, 2, 3, 4 ];
+}
```

Comment:
more easy and elegant :
```
Array.from({length: articles.length}, (elem, idx) => idx+1)
```
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/from